contents/docs/plugins/build/reference.md

```diff
@@ -452,8 +524,58 @@ export async function processEvent (event, { jobs }) {
 }
 ```
 
-## Limitations
+## Metrics
+
+> **Minimum PostHog version:** 1.27.0
+
+Plugin metrics allow plugin developers to provide metrics for users about plugin performance. They will appear in a chart on the plugins page, which is made visible by clicking the chart icon for a specific plugin. 
+
+This could be tracking the number of errors and successes when exporting events to another service, or the maximum amount of time taken for a request to a third-party API to complete, for example.
+### Specifying metrics
```

Comment:
Can/should we provide some basic metrics by default? For example, the time it took to execute a function? This is something we know.

Reply:
Hmm, so we're able to provide some metrics by default, but I'm trying to keep that to a minimum. It requires a bit of work on the plugin server and brings noise to users. Happy to take some suggestions on default metrics, but I think function execution time is maybe too general at this stage, and something one could implement with a few lines of JS if they wanted to

Reply:
True - it's likely very easy to add this. The counter argument is that because it's so easy we should do it for them. I can definitely see value in us flagging functions that are long-running and potentially causing problems or even timing out.

A middle ground could be to add the simple stuff to the plugin template.